sdk/core/core-amqp/api-extractor.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+  "mainEntryPointFilePath": "typings/src/index.d.ts",
+  "docModel": {
+    "enabled": false
+  },
+  "apiReport": {
+    "enabled": true,
+    "reportFolder": "./review"
+  },
+  "dtsRollup": {
+    "enabled": true,
+    "untrimmedFilePath": "",
+    "publicTrimmedFilePath": "./typings/latest/core-amqp.d.ts"
```

Comment:
They are being called "types" instead of "typings" for other packages.
Which one to pick?

Reply:
I'll switch to `types` since that's what it looks like most packages are doing.